ga4gh/client.py

```diff
@@ -16,12 +16,37 @@
 
 
 class HTTPClient(object):
-
-    workaroundGoogle = 'google'
-
     """
-    Simple HTTP client for the GA4GH protocol.
+    GA4GH Http Client
     """
+    workaroundGoogle = 'google'
+
+    class RunRequestContainer(object):
+        """
+        A container class for arguments to runRequest
+        """
+        def __init__(
+            self, request=None, url=None, protocolClass=None,
+                listAttr=None, requestMethod='post', id_=None):
+            self.request = request
+            self.url = url
+            self.protocolClass = protocolClass
+            self.listAttr = listAttr
+            self.requestMethod = requestMethod
+            self.id_ = id_
```

Comment:
Danny Colligan <notifications@github.com> writes:
> Let's just keep id_, since it keeps one from mindlessly writing id = something
> or self.id = id or some such and overriding the builtin id function.

  self.id_ = id

;-)

ident ?